src/Wallabag/CoreBundle/Helper/ContentProxy.php

```diff
@@ -22,21 +23,32 @@ class ContentProxy
     protected $ignoreOriginProcessor;
     protected $validator;
     protected $logger;
+    protected $container;
     protected $mimeTypes;
     protected $fetchingErrorMessage;
     protected $eventDispatcher;
     protected $storeArticleHeaders;
 
-    public function __construct(Graby $graby, RuleBasedTagger $tagger, RuleBasedIgnoreOriginProcessor $ignoreOriginProcessor, ValidatorInterface $validator, LoggerInterface $logger, $fetchingErrorMessage, $storeArticleHeaders = false)
+    private $clientSideRenderedSitesProxyUrl;
+    private $clientSideRenderedSitesProxyAll;
+    private $clientSideRenderedSitesToProxy;
+
+    public function __construct(Graby $graby, RuleBasedTagger $tagger, RuleBasedIgnoreOriginProcessor $ignoreOriginProcessor, ValidatorInterface $validator, LoggerInterface $logger, ContainerInterface $container, $fetchingErrorMessage, $storeArticleHeaders = false)
```

Comment:
Definitively not, never inject the container into a service.
If you want to inject parameters see https://symfony.com/doc/4.4/service_container.html#manually-wiring-arguments

Reply:
OK, fixed.